libs/application/templates/children-residence-change/src/fields/ContactInfo/index.tsx

```diff
@@ -30,22 +35,24 @@ const ContactInfo = ({ error, application, field }: FieldBaseProps) => {
       </Box>
       <Box marginTop={5}>
         <Controller
-          name="email"
-          defaultValue={getValue('contactInfo')?.[0]}
+          name={emailId}
+          defaultValue={getValue(phoneNumberId)}
           render={({ value, onChange }) => {
             return (
               <Input
-                id={`${id}[0]`}
-                name={`${id}[0]`}
+                ref={register}
+                id={emailId}
+                name={emailId}
                 backgroundColor="blue"
-                type="text"
+                type="email"
                 label={formatMessage(otherParent.inputs.emailLabel)}
                 value={value}
-                hasError={!!error}
+                hasError={!!emailError}
```

Comment:
Do we have to use not not here? Wondering if we know the type `undefined | string` and can then check if it is undefined or do we ever get an empty string?

Reply:
No we shouldn't get an empty string here. Always just undefined or some error value